Edge does not work on saucelabs

# Bug
Edge does not work on saucelabs; produces the following error:
`Caused by: org.openqa.selenium.WebDriverException: Misconfigured -- Unsupported OS/browser/version/device combo: OS: 'Windows 10', Browser: 'edge', Version: 'latest', Device: 'unspecified', Screen Resolution: '1600x1200' (WARNING: The server did not provide any stacktrace information)`

# Expected
It should start a Win10/Edge run on saucelabs

# Repro
1. Set `desired-capabilities.browser-type=edge`
2. Run any test
3. See error

### Fix
change `EDGE("Edge"),` to `EDGE("MicrosoftEdge"),` in src/main/java/io/github/kgress/scaffold/models/enums/BrowserType.java

--- a/models/src/main/java/io/github/kgress/scaffold/models/enums/BrowserType.java
+++ b/models/src/main/java/io/github/kgress/scaffold/models/enums/BrowserType.java
@@ -9,7 +9,7 @@ public enum BrowserType {
     CHROME("Chrome"),
     SAFARI("Safari"),
     INTERNET_EXPLORER("Internet Explorer"),
-    EDGE("Edge"),
+    EDGE("MicrosoftEdge"),
     OPERA("Opera");
 
     private String browserName;

